src/components/Sidebar/components/SideBarItem/SideBarItem.tsx

```diff
@@ -1,11 +1,28 @@
 import React, { ReactNode } from "react";
-import "styled-components/macro";
+import styled from "styled-components/macro";
 import { Color } from "variables";
 
 type Props = {
   children: ReactNode;
 };
 
+const Progress = styled.div`
+  background-color: #24292e;
+  color: white;
+  display: table-cell;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 14px;
+  line-height: 21px;
+  height: 25px;
+  width: 200px;
+  padding: 11px 15px;
+  vertical-align: middle;
+
+  &:hover {
+    background-color: #0366d6;
```

Comment:
I think I forgot to mention this color in the issue! sorry for that.
```suggestion
    background-color: ${Color.DodgerBlue};
```